src/test/java/apicurito/tests/configuration/templates/ApicuritoTemplate.java

```diff
@@ -35,14 +39,38 @@ public static Template getTemplate() {
         }
     }
 
-    public static void setInputStreams() {
+    public static Deployment getOperatorDeployment() {
```

Comment:
this is used only when operator image was specified and custom deployment should be generated - use more appropriate name like getUpdatedOperatorDeployment()